modules/power-monitoring-installing-pmo.adoc

```diff
@@ -8,18 +8,13 @@
 
 As a cluster administrator, you can install the {PM-operator} from OperatorHub by using the {product-title} web console.
 
-[WARNING]
-====
-You must remove any previously installed versions of the {PM-operator} before installation.
-====
-
```

Comment:
How about we retain this since it is the most safest way to ensure a clean install? 
cc: @vprashar2929 @vimalk78